.github/actions/rockylinux-build/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM rockylinux:8
+FROM rockylinux:8 As rockylinux8
```

```suggestion
FROM rockylinux:8
```

not really needed when you are not doing a multi-stage build